nh_eobs_mental_health/models/nh_eobs_ward_dashboard.py

```diff
@@ -31,7 +31,13 @@ class NHClinicalWardDashboard(orm.Model):
         'capacity_count': fields.integer('Free Beds'),
         'workload_count': fields.integer('All patients associated with ward'),
         'on_ward_count': fields.integer(
-            'All patients in bed and not on Patient Monitoring Exception')
+            'All patients in bed and not on Patient Monitoring Exception'),
+        'refused_obs_count': fields.integer(
+            'All patients currently refusing observations')
+    }
+
+    _defaults = {
+        'refused_obs_count': 0
     }
```

Comment:
This is my bad, I believe I added this during development just to get something displaying on the screen. No need for it to be there though and could potentially be misleading if a bug causes the index card to be 0 when there are actually refused patients.